internal/manifestvalues/flags/use_default_options.go

```diff
@@ -0,0 +1,14 @@
+package flags
+
+import "github.com/spf13/cobra"
+
+type UseDefaultOptions struct {
+	UseDefault []string
+}
+
+func (opts *UseDefaultOptions) AddFlagsToCommand(cmd *cobra.Command) {
+	flags := cmd.Flags()
+	flags.StringArrayVar(&opts.UseDefault, "use-default", opts.UseDefault,
+		"Instruct glasskube to use the default value for the speciefied definition name(s).\n"+
+			"Specify \"all\" to use all available default values.")
```

Comment:
I don't think the pflag library supports this for a StringArray type flag. IMO the current behavior is OK, i.e. print help and exit (rather than silently do nothing), but an alternative would be providing two flags (in which case the handling of "all" can be removed), either: 
- `--use-defaults` (Bool) and `--use-default-for` (StringArray)
- `--use-default-all` and `--use-default`